messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistModalActivity.kt

```diff
@@ -61,6 +62,12 @@ class GistModalActivity : AppCompatActivity(), GistListener, GistViewListener {
         } ?: run {
             finish()
         }
+
+        // Disable back button for persistent messages
```

Comment:
Might need more clarification here, something like this?
```suggestion
        // Add back button ti handle in-app message behavior, enabled false with disable back press for persistent messages
```